internal/cli/trigger_apply.go

```diff
@@ -311,12 +313,17 @@ func (c *TriggerApplyCommand) Run(args []string) int {
 		return 1
 	}
 
-	c.ui.Output("Trigger %q (%s) has been %s", resp.Trigger.Name, resp.Trigger.Id,
+	c.ui.Output("Trigger %q (%s) has been %s\n", resp.Trigger.Name, resp.Trigger.Id,
 		action, terminal.WithSuccessStyle())
 
-	c.ui.Output("Trigger ID: %s", resp.Trigger.Id, terminal.WithSuccessStyle())
-	// TODO(briancain): update output to show trigger URL with wp server attached once http service is implemented
-	//c.ui.Output("Trigger URL: %s", resp.TriggerURL, terminal.WithSuccessStyle())
+	triggerID := resp.Trigger.Id
+	addr := strings.Split(c.clientContext.Server.Address, ":")[0]
+	port := serverconfig.DefaultHTTPPort
+	serverAddr := fmt.Sprintf("%s:%s", addr, port)
+	serverTriggerURL := fmt.Sprintf("https://%s/v1/trigger/%s", serverAddr, triggerID)
+
+	c.ui.Output(" Trigger ID: %s", triggerID, terminal.WithSuccessStyle())
```

Comment:
```suggestion
	c.ui.Output("Trigger ID: %s", triggerID, terminal.WithSuccessStyle())
```

Not sure if the leading space is intentional here

Reply:
It is! I tried aligning the two on the “:” here, similar to how waypoint up aligns the ui and server host name. It looks less weird on the command line, but I’m ok to delete the space if it’s not worth it

Reply:
👍 if it's intentional then let's just leave it